### ISSUE TITLE
Cannot Start The application .

### ISSUE THREAD

**Issue by aashay-accolite** [angular/material-start#163]
I have install all the modules using npm and installed jspm.
All the files are in the respective folders.
Still not able to run the application.
Can somebody please help me ?
![angularmatterialstart](https://cloud.githubusercontent.com/assets/25149877/22370701/bad9a0e0-e4b8-11e6-86cd-e1532f7e125b.PNG)

**Comment by Phoqe** [comment]
Can confirm, getting this with a fresh clone.

**Comment by jose96043** [comment]
Having same issue. Is this an issue with current master branch?

```javascript
Error: (SystemJS) [BABEL] unknown: Unknown option: direct.0
	ReferenceError: [BABEL] unknown: Unknown option: direct.0
	    at Logger.error (http://127.0.0.1:8080/jspm_packages/npm/babel-core@5.8.38/browser.js:8955:11)
	    at OptionManager.mergeOptions (http://127.0.0.1:8080/jspm_packages/npm/babel-core@5.8.38/browser.js:9463:29)
	    at OptionManager.init (http://127.0.0.1:8080/jspm_packages/npm/babel-core@5.8.38/browser.js:9553:10)
	    at File.initOptions (http://127.0.0.1:8080/jspm_packages/npm/babel-core@5.8.38/browser.js:8193:75)
	    at new File (http://127.0.0.1:8080/jspm_packages/npm/babel-core@5.8.38/browser.js:8183:22)
	    at Pipeline.transform (http://127.0.0.1:8080/jspm_packages/npm/babel-core@5.8.38/browser.js:13515:16)
	Error loading http://127.0.0.1:8080/src/boot/boot.js
	    at Logger.error (http://127.0.0.1:8080/jspm_packages/npm/babel-core@5.8.38/browser.js:8955:11)
	    at OptionManager.mergeOptions (http://127.0.0.1:8080/jspm_packages/npm/babel-core@5.8.38/browser.js:9463:29)
	    at OptionManager.init (http://127.0.0.1:8080/jspm_packages/npm/babel-core@5.8.38/browser.js:9553:10)
	    at File.initOptions (http://127.0.0.1:8080/jspm_packages/npm/babel-core@5.8.38/browser.js:8193:75)
	    at new File (http://127.0.0.1:8080/jspm_packages/npm/babel-core@5.8.38/browser.js:8183:22)
	    at Pipeline.transform (http://127.0.0.1:8080/jspm_packages/npm/babel-core@5.8.38/browser.js:13515:16)
	Error loading http://127.0.0.1:8080/src/boot/boot.js
```

**Comment by Ronin11** [comment]
Also getting this with a fresh master branch.

**Comment by angelomichel** [comment]
Can you validate that the code is not babelified as well? 

The error caused (on my machine) because it didn't went through babel.

Go to: http://127.0.0.1:8080/src/boot/boot.js   
This url should **not** start with:

`import angular from 'angular';
...`

**Comment by angelomichel** [comment]
I've committed a fix and asked for a pull request: https://github.com/angular/material-start/pull/165 

This should fix it if you re-run `npm i` and `jspm update`.

**Comment by dave-mitchell** [comment]
This was an issue for me in any other branch/tag besides es6-tutorial.  Even step-10-finished was broken, giving me this message about importing babel-core.5.8.38.  I was able to resolve the issue by copying the es6-tutorial config.js into the other branches as I was working, as that was the only file I could find the string 5.8.38 in.  Hope that helps.

**Comment by ThomasBurleson** [comment]
@EladBezalel - Can you investigate please.

**Comment by checklist** [comment]
@dave-mitchell got it to work for me. This is the file you need: https://github.com/angular/material-start/blob/es6-tutorial/app/config.js

**Comment by EladBezalel** [comment]
@checklist can you do a pr please?

**Comment by Splaktar** [comment]
This was fixed by switching to traceur in 7792fa091bffff90f52389abf031c77cb214aaac.
